eng/versioning/version_client.txt

```diff
@@ -133,17 +133,17 @@ com.azure:azure-sdk-template-two;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-sdk-template-three;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-spring-data-cosmos;3.21.0;3.22.0-beta.1
 com.azure:azure-spring-data-cosmos-test;3.0.0-beta.1;3.0.0-beta.1
-com.azure:azure-storage-blob;12.16.1;12.17.0-beta.2
-com.azure:azure-storage-blob-batch;12.12.2;12.13.0-beta.2
-com.azure:azure-storage-blob-changefeed;12.0.0-beta.18;12.0.0-beta.19
-com.azure:azure-storage-blob-cryptography;12.15.2;12.16.0-beta.2
-com.azure:azure-storage-blob-nio;12.0.0-beta.19;12.0.0-beta.20
-com.azure:azure-storage-common;12.15.2;12.16.0-beta.2
-com.azure:azure-storage-file-share;12.12.2;12.13.0-beta.2
-com.azure:azure-storage-file-datalake;12.9.1;12.10.0-beta.2
-com.azure:azure-storage-internal-avro;12.2.2;12.3.0-beta.2
-com.azure:azure-storage-perf;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-storage-queue;12.12.2;12.13.0-beta.2
+com.azure:azure-storage-blob;12.16.1;12.17.0
+com.azure:azure-storage-blob-batch;12.12.2;12.13.0
+com.azure:azure-storage-blob-changefeed;12.0.0-beta.18;12.0.0
+com.azure:azure-storage-blob-cryptography;12.15.2;12.16.0
+com.azure:azure-storage-blob-nio;12.0.0-beta.19;12.0.0
+com.azure:azure-storage-common;12.15.2;12.16.0
+com.azure:azure-storage-file-share;12.12.2;12.13.0
+com.azure:azure-storage-file-datalake;12.9.1;12.10.0
+com.azure:azure-storage-internal-avro;12.2.2;12.3.0
+com.azure:azure-storage-perf;1.0.0-beta.1;1.0.0
```

Comment:
This looks suspect to me, I think we need to revert this line.

Reply:
yes we should revert this one. we don't release this library